app/containers/Footer/styledComponents/wrapper.js

```diff
@@ -0,0 +1,15 @@
+import styled from 'styled-components';
```

Comment:
- Do not create styleComponents subfolder. Add components directly to component folder.
- Components file names start with capital later